app/src/main/kotlin/at/bitfire/davdroid/syncadapter/SyncManager.kt

```diff
@@ -326,6 +320,25 @@ abstract class SyncManager<ResourceType: LocalResource<*>, out CollectionType: L
         })
     }
 
+    private fun logSyncTime() {
+        val serviceType = when (authority) {
+            CalendarContract.AUTHORITY -> Service.TYPE_CALDAV
+            ContactsContract.AUTHORITY,
+            context.getString(R.string.address_books_authority) -> Service.TYPE_CARDDAV
+            else -> throw IllegalArgumentException("Invalid authority")
```

Comment:
This line crashes on tasks sync (we don't have the task authorities taken into account.)

Reply:
I am going to leave it as is, for now then

Reply:
Then DAVx5 crashes as soon as there is a tasks app installed ;)

I suggest to map the contacts/address book authority to `Service.TYPE_CARDDAV` and anything else to CalDAV.

Reply:
> I'll think about alternatives… really hard.

I have made the change and rebased with main-ose. We could leave it like this for now. And change it in the future?